sources/model/Cloud.cpp

```diff
@@ -51,6 +78,9 @@ Cloud::~Cloud()
 
     if (channelMults)
         delete[] channelMults;
+    for (int i = 0; i < g_maxMidiVoices; i++){
+        delete &playedCloudMidi[i];
```

Comment:
Destruction du mauvais pointeur ici.
```suggestion
        delete playedCloudMidi[i];
```

Reply:
vu